v2/cmd/controller/app/setup.go

```diff
@@ -37,6 +37,7 @@ import (
 	. "github.com/Azure/azure-service-operator/v2/internal/logging"
 	asometrics "github.com/Azure/azure-service-operator/v2/internal/metrics"
 	armreconciler "github.com/Azure/azure-service-operator/v2/internal/reconcilers/arm"
+	"github.com/Azure/azure-service-operator/v2/pkg/common"
```

Comment:
I'll beat @matthchr to this (because he always catches me out) - move this into the import group for local packages, just below.